projects/1.20/assets/refurbished-furniture/refurbished_furniture/lang/zh_cn.json

```
@@ -0,0 +1,646 @@
+{
+    "itemGroup.refurbished_furniture": "MrCrayfish的家具模组",
+    "filterCategory.refurbished_furniture.general": "常规",
+    "filterCategory.refurbished_furniture.general.desc": "所有基础家具。椅子、桌子等等",

```

```suggestion
    "filterCategory.refurbished_furniture.general.desc": "所有基础家具，椅子、桌子等等",
```
为改硬改的产物，后方如有键中句号键尾无句号的情况建议同样更改